node/src/chainspec/mainnet.rs

```diff
@@ -53,6 +88,66 @@ fn generate_session_keys(
 	tangle_runtime::opaque::SessionKeys { grandpa, babe, im_online }
 }
 
+pub fn local_mainnet_config(chain_id: u64) -> Result<ChainSpec, String> {
+	let wasm_binary = WASM_BINARY.ok_or_else(|| "tangle wasm not available".to_string())?;
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenDecimals".into(), 18u32.into());
+	properties.insert("ss58Format".into(), 4006.into());
+	// TODO: use mainnet ss58Format
+	properties.insert("ss58Format".into(), 42.into());
```

Comment:
repeat of line 95

Reply:
also missing token symbol, both here and in `tangle_mainnet_config`

Reply:
Yup, that was a merege commit maybe.